.github/workflows/dotnetcore.yml

```
@@ -0,0 +1,25 @@
+name: .NET Core
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
```

I do my dev on windows, so might as well CI on windows.

```suggestion
    runs-on: windows-latest
```